spec/unit/mutest/integration_spec.rb

```diff
@@ -11,14 +11,15 @@
   end
 
   describe '.setup' do
-    subject { described_class.setup(kernel, name) }
+    subject { described_class.setup(requirer, name) }
 
-    let(:name)   { 'null'               }
-    let(:kernel) { class_double(Kernel) }
+    let(:name) { 'null' }
+    let(:requirer) { Mutest::Requirer.new }
 
     before do
-      expect(kernel).to receive(:require)
+      expect(requirer).to receive(:require)
         .with('mutest/integration/null')
+        .and_call_original
```

Comment:
One solution would be to inject a different requiring in this test that just invokes some form of `File.exist?`. I wouldn't do that but it seems acceptable and would enforce the check you are saying is useful